roles/audit/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: configure audisp syslog plugin
+  template:
+    src: ../templates/etc/audisp/plugins.d/syslog.conf
```

Comment:
templates are discovered by default in the templates dir, so you just need:
`etc/audisp/plugins.d/syslog.conf`